gcpy/benchmark/modules/benchmark_models_vs_ebas_o3.py

```diff
@@ -727,7 +562,7 @@ def plot_single_station(
         marker='^',
         markersize=4,
         lw=1,
-        label='Observations'
+        label='Surface O3 (EBAS, 2019)'
```

Comment:
This is a noteworthy change since the plots look different. How frequently will this source change? Do you think it should be set in the config file rather than in this file?

Reply:
@lizziel, we could set it in the config file.  The only thing is that the code that we received is fairly specific to the type of data.  Although all we would have to do to make it more general is to add another reading routine for a different type of data.

Reply:
Now resolved in 567342f